### PR TITLE
Spacing system - content rewrite

### PR DESCRIPTION
**Pull request by chri2547**
image

### CODE REVIEW

File: _docs/style/spacing.md [hunk 1]
```diff
@@ -3,59 +3,75 @@ title: Spacing
...
 
 <section class="static-section" markdown="1">
 
-## Introduction
 <div class="hxRow" markdown="1">
 
 {% column left:"hxCol hxSpan-12-xs hxSpan-12-sm hxSpan-12-md hxSpan-12-lg" %}
 
-The Helix spacing system was influenced by [Priyanka Godbole’s article](https://blog.prototypr.io/a-framework-for-creating-a-predictable-and-harmonious-spacing-system-8eee8aaf773c) in which she outlines steps for defining a system based on the **rule of three C’s**.
+The document provides guidelines you should use when deciding on spacing values between and among user interface components. Specific values are provided, with justification for why we recommend certain spacing values.
```
[the remark this answers is not in the document]
Comment (chri2547) [reply _docs/style/spacing.md]: Done.

File: _docs/style/spacing.md [hunk 2]
```diff
@@ -3,59 +3,75 @@ title: Spacing
...
+
+<div class="hxRow" markdown="1">
+
+{% column left:"hxCol hxSpan-12-xs hxSpan-12-sm hxSpan-12-md hxSpan-12-lg" %}
+
+## Best practices
+
+Consider the following best practices when you apply the Helix spacing system:
+
+- In most cases, use medium spacing as the default.
```
[the remark this answers is not in the document]
Comment (chri2547) [reply _docs/style/spacing.md]: oh yes, good idea. done.

File: _docs/style/spacing.md [hunk 3]
```diff
@@ -3,59 +3,75 @@ title: Spacing
...
+- If you intend to use a different spacing value, keep in mind that large spacing can result in too much whitespace, and small spacing can be too data dense and difficult for the user to parse.
```
[the remark this answers is not in the document]
Comment (chri2547) [reply _docs/style/spacing.md]: Done.

File: _docs/style/spacing.md [hunk 4]
```diff
@@ -3,59 +3,75 @@ title: Spacing
...
+- Don't mix spacing sizes within a component.
```
[the remark this answers is not in the document]
Comment (chri2547) [reply _docs/style/spacing.md]: Done.

File: _docs/style/spacing.md [hunk 5]
```diff
@@ -3,59 +3,75 @@ title: Spacing
...
 
-## Container
 <div class="hxRow" markdown="1">
 
 {% column left:"hxCol hxSpan-12-xs hxSpan-12-sm hxSpan-12-md hxSpan-12-lg" %}
 
-Pages, cards, modals, panes, etc. are containers that frame content within a UI. Containers relate directly to the canvas / grid to create the layout of any page. For this reason, we defined a 20px-square inset inside all containers to be harmonious with our column gutters.
+## Container
+
+Pages, cards, modals, panes, and choice tiles are containers that frame content within a UI. By default, apply a square inset of 20px to the top, right, bottom, and left side of the container. The 20px is the medium spacing value and you can use smaller or larger spacing, if required.
```
[the remark this answers is not in the document]
Comment (chri2547) [reply _docs/style/spacing.md]: Done.

File: _docs/style/spacing.md [hunk 6]
```diff
@@ -142,29 +156,31 @@ Line-heights across all font-sizes must be proportional to each other to create
...
 <div class="hxRow"  markdown="1">
 
 {% column left:"hxCol hxSpan-12-xs hxSpan-12-sm hxSpan-4-md hxSpan-4-lg" %}
 
+### Headers
 
-All content is stacked vertically, yet using line-height values alone does not convey enough visual hierarchy and can result in overly dense layouts.
-
-Priyanka’s framework provides a solution for this using header and leaf-node margins. Following her example, we experimented with different spacing values, in several layouts, and found that a single spacing value of eight pixels worked best for all header margins.
+Applying line height values to vertically stacked content can still result in overly dense layout. Helix user testing revealed that participants preferred adding an extra 8px between stacked elements.
```
[the remark this answers is not in the document]
Comment (chri2547) [reply _docs/style/spacing.md]: Woot!

File: _docs/style/spacing.md [hunk 7]
```diff
@@ -173,9 +189,9 @@ Priyanka’s framework provides a solution for this using header and leaf-node m
 
 {% column left:"hxCol hxSpan-12" %}
 
-### Leaf-nodes
+### Leaf nodes
 
-Next we defined spacing values for leaf-node content. Unlike header margins, it is impossible to use a single spacing value across all types of leaf-node content. For each type of leaf-node content, the spacing is as follows:
+A leaf node includes any content that is not a header, including paragraphs, form elements, list items, and table cells. There is no single spacing value that can be used across all leaf node content. Based on Helix user testing, we have determined the following spacing for leaf node content:
```
[the remark this answers is not in the document]
Comment (chri2547) [reply _docs/style/spacing.md]: Beauty!

File: _docs/style/spacing.md [hunk 8]
```diff
@@ -267,56 +298,73 @@ We found that eight pixels of vertical space between consecutive, stacked inputs
...
 <section class="static-section" markdown="1">
+
 <div class="hxRow"  markdown="1">
 
 {% column left:"hxCol hxSpan-12" %}
 
 ## Component
 
-Components are buttons, input fields, icons, and so on. Components are often placed next to each other horizontally (inline). All components sizing and spacing is defined in multiples of four.
+Components such as buttons, text inputs, and date pickers are often placed beside each other on a control panel page. The Helix spacing system uses multiples of 4px as the baseline spacing value.
```
[the remark this answers is not in the document]
Comment (chri2547) [reply _docs/style/spacing.md]: noice!

File: _docs/style/spacing.md [hunk 9]
```diff
@@ -267,56 +298,73 @@ We found that eight pixels of vertical space between consecutive, stacked inputs
...
 
-For consistency, eight pixels was defined for left and right padding inside components.
+**Spacing between components**: Use 8px spacing between adjacent components. For a tighter association between components, use 4px.
 
-#### Spacing between components
+**Outstanding questions**
 
-We used a simple rule of eight pixels of space between any two adjacent components for the majority of applications. However, we decided to allow the use of four pixels to show a tighter association between components.
+- when would the 4px spacing be used? When is a tighter association needed?
+- Is there any other content that we should include?
```
[the remark this answers is not in the document]
Comment (chri2547) [reply _docs/style/spacing.md]: I've made a note. Will add to jira ticket.

File: _docs/style/spacing.md (hunk 7 above)
[the remark this answers is not in the document]
Comment (chri2547) [reply _docs/style/spacing.md]: Replace leaf node with "Body elements"

File: _docs/style/spacing.md [hunk 10]
```diff
@@ -267,56 +298,73 @@ We found that eight pixels of vertical space between consecutive, stacked inputs
...
+**Spacing inside components**: Inside a component, use 8px spacing for the left and right padding.
 
-For consistency, eight pixels was defined for left and right padding inside components.
+**Spacing between components**: Use 8px spacing between adjacent components. For a tighter association between components, use 4px.
 
-#### Spacing between components
+**Outstanding questions**
 
-We used a simple rule of eight pixels of space between any two adjacent components for the majority of applications. However, we decided to allow the use of four pixels to show a tighter association between components.
+- when would the 4px spacing be used? When is a tighter association needed?
```
[the remark this answers is not in the document]
Comment (chri2547) [reply _docs/style/spacing.md]: Ok, wordsmithed this one. Done.

File: _docs/style/spacing.md [hunk 11]
```diff
@@ -3,59 +3,83 @@ title: Spacing
 layout: docs
 category: Style
 usage: >
-  Helix spacing system provides a logical set of values and limited application rules to better enable design and development teams to use spacing deliberately for improved readability and consistency in product.
+  The Helix spacing system provides a set of values you can use when you determine the amount of space between user interface (UI) components. By applying these spacing values consistently you adhere to accessibility guidelines and improve the readability of control panel pages.
```
Comment (palindromebart) [inline _docs/style/spacing.md]: The Helix spacing system provides a set of values you can use to keep consistent spacing of user interface (UI) components. By applying these spacing values consistently, you adhere to Helix accessibility guidelines and can improve the readability of control panel pages.

File: _docs/style/spacing.md [hunk 12]
```diff
@@ -3,59 +3,83 @@ title: Spacing
...
 
 <section class="static-section" markdown="1">
 
-## Introduction
 <div class="hxRow" markdown="1">
 
 {% column left:"hxCol hxSpan-12-xs hxSpan-12-sm hxSpan-12-md hxSpan-12-lg" %}
 
-The Helix spacing system was influenced by [Priyanka Godbole’s article](https://blog.prototypr.io/a-framework-for-creating-a-predictable-and-harmonious-spacing-system-8eee8aaf773c) in which she outlines steps for defining a system based on the **rule of three C’s**.
+The document provides guidelines you should use when deciding on spacing values between and among user interface components. Specific values are provided, with justification for why we recommend them.
```
Comment (palindromebart) [inline _docs/style/spacing.md]: is this paragraph necessary? i suggest we cut it.

File: _docs/style/spacing.md [hunk 13]
```diff
@@ -3,59 +3,83 @@ title: Spacing
...
+
+{% column left:"hxCol hxSpan-12-xs hxSpan-12-sm hxSpan-12-md hxSpan-12-lg" %}
+
+## Best practices
+
+Consider the following best practices when you apply the Helix spacing system:
+
+- In most cases, use medium spacing (20px) as the default.
+- When choosing spacing values for a new design, consider the user's needs and use case. Use small spacing when users prefer greater data density. Use large spacing to increase whitespace, which can help enhance user comprehension.
+- Avoid uneven spacing (padding) on containers. In the majority of cases, containers must use square Inset spacing, which means that the top, right, bottom, and left spacing values are equal inside the container.
```
Comment (palindromebart) [inline _docs/style/spacing.md]: capitalization?

File: _docs/style/spacing.md [hunk 14]
```diff
@@ -3,59 +3,83 @@ title: Spacing
...
 
-## Container
 <div class="hxRow" markdown="1">
 
 {% column left:"hxCol hxSpan-12-xs hxSpan-12-sm hxSpan-12-md hxSpan-12-lg" %}
 
-Pages, cards, modals, panes, etc. are containers that frame content within a UI. Containers relate directly to the canvas / grid to create the layout of any page. For this reason, we defined a 20px-square inset inside all containers to be harmonious with our column gutters.
+## Container spacing
+
+Pages, cards, modals, panels, and choice tiles are examples of containers that frame content within a UI. By default, apply a square inset of 20px to the top, right, bottom, and left side of the container. The 20px is the medium spacing value and you can use smaller or larger spacing, if required.
```
Comment (palindromebart) [inline _docs/style/spacing.md]: "Choice Tiles" needs to be changed to "Option Tiles" throughout this doc.

File: _docs/style/spacing.md [hunk 15]
```diff
@@ -3,59 +3,83 @@ title: Spacing
...
+
 <div class="hxRow"  markdown="1">
 
 {% column left:"hxCol hxSpan-12-xs hxSpan-12-sm hxSpan-4-md hxSpan-4-lg" %}
 
-### Container Spacing Variants
+### Container spacing variants
 
-Some cases may require different spacing to accommodate very small or large containers (see [choice-tiles]({{site.url}}/components/choice-tiles.html)). Therefore we have included Small, Medium, and Large container spacing sizes to facilitate your content.
+In some cases you may be required to use smaller or larger spacing than the 20px default value. Use the following spacing values to apply extra-small through extra-extra-large spacing. Keep in mind that large spacing can result in too much whitespace while small spacing can lead to a data dense UI, which can be difficult for a user to process.
```
Comment (palindromebart) [inline _docs/style/spacing.md]: we only have small, medium and large Option Tiles now.
Small: 12px
Medium: 20px
Large: 32px

Comment (palindromebart) [reply _docs/style/spacing.md]: also, in the visual we don't need the redlines.

File: _docs/style/spacing.md [hunk 16]
```diff
@@ -85,86 +110,70 @@ Exceptions are made for some elements such as dashboard panels and [cards]({{sit
...
+
+<section class="static-section" markdown="1">
 
 <div class="hxRow"  markdown="1">
 
 {% column left:"hxCol hxSpan-12-xs hxSpan-12-sm hxSpan-4-md hxSpan-4-lg" %}
 
-**Baseline grid is multiples of four.**
-Just like using columns as guides in your design, the baseline grid is a way to help you make decisions and build consistency into your layout based on multiples of four.
+**Baseline value**: The Helix spacing system uses four as the baseline number on which all other spatial values are calculated. For example, the font size value for all headings are multiples of four. The baseline value helps you build consistency into your layout.
```
Comment (palindromebart) [inline _docs/style/spacing.md]: the text "four" confused me at first. can we use the number instead? or in addition?

File: _docs/style/spacing.md (hunk 11 above)
Comment (palindromebart) [inline _docs/style/spacing.md]: @racksmangonzalez The visuals throughout need to have the grayline labels standardized for line length.

File: _docs/style/spacing.md [hunk 17]
```diff
@@ -173,150 +182,180 @@ Priyanka’s framework provides a solution for this using header and leaf-node m
...
+</section>
+
+<section class="static-section" markdown="1">
+
+<div class="hxRow" markdown="1">
 
 {% column left:"hxCol hxSpan-12-xs hxSpan-12-sm hxSpan-4-md hxSpan-4-lg" %}
 
-#### Paragraphs
+**Spacing inside components**: Within a component, use 8px spacing for the left and right padding.
```
Comment (palindromebart) [inline _docs/style/spacing.md]: is this correct?  it doesn't match the visual... but i cant remember...
let/s just double check the values.

also, the visuals here really need to match what you mention in the examples. so we shoudl include two tightly coupled inputs with 4 px between them in the visual

File: _docs/style/spacing.md (hunk 11 above)
Comment (chri2547) [reply _docs/style/spacing.md]: Done.

File: _docs/style/spacing.md (hunk 12 above)
Comment (chri2547) [reply _docs/style/spacing.md]: yah, agree. removed it.

File: _docs/style/spacing.md (hunk 13 above)
Comment (chri2547) [reply _docs/style/spacing.md]: good catch! done.

File: _docs/style/spacing.md (hunk 14 above)
Comment (chri2547) [reply _docs/style/spacing.md]: Done.

File: _docs/style/spacing.md (hunk 15 above)
Comment (chri2547) [reply _docs/style/spacing.md]: Done and done!

File: _docs/style/spacing.md (hunk 16 above)
Comment (chri2547) [reply _docs/style/spacing.md]: sure, switched to 4.

File: _docs/style/spacing.md (hunk 17 above)
Comment (chri2547) [reply _docs/style/spacing.md]: Comment 1: asked in content channel whether it's 8px or 12px.
Comment 2: asked roman to address this.

File: _docs/style/spacing.md (hunk 11 above)
Comment (chri2547) [reply _docs/style/spacing.md]: Done.

File: _docs/style/spacing.md (hunk 9 above)
[the remark this answers is not in the document]
Comment (chri2547) [reply _docs/style/spacing.md]: 4px visuals have been added.

**Comment by cyrichardson** [comment]
@chri2547 On it.

**Comment by chri2547** [comment]
Spacing visuals updated, editorial feedback incorporated. On hold until roman audits all inputs for internal spacing specs.

File: _docs/style/spacing.md [hunk 18]
```diff
@@ -85,86 +105,70 @@ Exceptions are made for some elements such as dashboard panels and [cards]({{sit
...
-## Content text
+Content is any information displayed inside a container. When you vertically stack content components, ensure that you conform to line height and bottom margin spacing values. Bottom margins contain extra spacing that improves visually perceptible intervals, which is helpful for conveying hierarchy.
 
-Content lives inside the container. Content is stacked vertically using line-heights and bottom margins (depending on type of content). Bottom margins add extra spacing that improves visually perceptible intervals, which is helpful for conveying hierarchy. Content contains:
+There are two types of content:
 
-- Headers are at the highest level of hierarchy for content (page title, section title, sub-section title, container title)
-- Just below the header hierarchy is the data interspersed in the form of paragraphs, lists, forms, and tables that we will refer to as leaf-nodes.
+- **Headers**: Headers are the highest level of the content hierarchy and are used for page titles, section titles, subsection titles, and  container titles.
+- **Body elements**: Body elements are located just below the header and consist of list items, drop-down selectors, and paragraphs.
```
Comment (palindromebart) [inline _docs/style/spacing.md]: these aren't the ONLY things. this is just an example list...
could include visualizations, cards, form controls, and body text.

maybe we should generalize to example categories. 
 Body elements are located below the header and consist of a wide variety of components including body text, list items, form controls, and visualizations.

Comment (palindromebart) [reply _docs/style/spacing.md]: LEt's bundle this into a follow up story

Comment (chri2547) [reply _docs/style/spacing.md]: Done.

File: _docs/style/spacing.md [hunk 19]
```diff
@@ -3,80 +3,100 @@ title: Spacing
...
+<section class="static-section" markdown="1">
+
+<div class="hxRow" markdown="1">
+
+{% column left:"hxCol hxSpan-12-xs hxSpan-12-sm hxSpan-12-md hxSpan-12-lg" %}
+
+## Best practices
 
-This framework enabled us to define the spacing system using a limited set of values and rules that facilitate information hierarchy and adhere to accessibility guidelines.
+Consider the following best practices when you apply the Helix spacing system:
```
[the remark this answers is not in the document]
Comment (chri2547) [reply _docs/style/spacing.md]: Done.

**Pull request merged**